src/main/java/net/simpvp/Misc/PotionEffectListener.java

```diff
@@ -0,0 +1,28 @@
+package net.simpvp.Misc;
+
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.entity.EntityPotionEffectEvent;
+import org.bukkit.potion.PotionEffectType;
+
+//This file disables elder guardians from giving mining fatigue within 100 blocks of spawn
+public class PotionEffectListener implements Listener {
+	
+	@EventHandler
+	public void potionEffectEvent(EntityPotionEffectEvent event) {
+		
+		//Check if world name is in the config
+		if (!Misc.instance.getConfig().getList("disableElderGuardians").contains(event.getEntity().getWorld().getName())) {

```

Comment:
I think getting the list is a bit more expensive than keeping it in a local variable. Could you load the config list into a private HashSet in the class when initializing the plugin?